website/docs/r/cse_first_seen_rule.html.markdown

```diff
@@ -0,0 +1,72 @@
+---
+layout: "sumologic"
+page_title: "SumoLogic: sumologic_cse_first_seen_rule"
+description: |-
+  Provides a CSE First Seen Rule
+---
+
+# sumologic_cse_first_seen_rule
+Provides a Sumo Logic CSE [First Seen Rule](https://help.sumologic.com/docs/cse/rules/write-first-seen-rule/).
+
+## Example Usage
+```hcl
+resource "sumologic_cse_first_seen_rule" "first_seen_rule" {
+  baseline_type          = "PER_ENTITY"
+  baseline_window_size   = "35000"
+  description_expression = "First User Login - {{ user_username }}"
+  enabled                = true
+
+  entity_selectors {
+        entity_type = "_username"
+        expression = "user_username"
+  }
+
+  entity_selectors {
+        entity_type = "_hostname"
+        expression = "dstDevice_hostname"
+  }
+
+  filter_expression     = "objectType=\"Network\""
+  group_by_fields       = ["user_username"]
+  is_prototype          = false
+  name                  = "First User Login"
+  name_expression       = "First User Login - {{ user_username }}"
+  retention_window_size = "86400000"
+  severity              = 1
+  value_fields          = ["dstDevice_hostname"]
+}
+```
+
+## Argument reference
+
+The following arguments are supported:
+
+- `baseline_type` - (Required) The baseline type. Current acceptable values are GLOBAL or PER_ENTITY
+- `baseline_window_size` - (Optional) The baseline window size in milliseconds 
```

Comment:
Now that you've made the window sizes required and removed the category in the code, we need to do it in the doc too.